Client/Ecosia/UI/NTP/ClimateImpactCounter/ArchProgressView.swift

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+
+struct RectangularArcShape: Shape {
+    var progress: CGFloat // Progress value between 0 and 1
+    
+    func path(in rect: CGRect) -> Path {
+        var path = Path()
+        
+        // Calculate the start and end angles for the arc
+        let startAngle = Angle(degrees: 180)
+        let endAngle = Angle(degrees: 180 + Double(180 * progress))
+        
+        // Create an arc based on the rectangle's dimensions
+        path.addArc(center: CGPoint(x: rect.midX, y: rect.maxY),  // Center at the bottom of the rect
+                    radius: min(rect.width, rect.height) / 1.5,     // Use the smaller dimension for the radius
+                    startAngle: startAngle,
+                    endAngle: endAngle,
+                    clockwise: false)
+        
+        return path
+    }
+}
+
+struct ArchProgressView: View {
```

Comment:
I see in this file there is both `Arch` and `Arc`. Afaik they can mean the same thing, but can we keep consistency and use `Arc` since that is also what the native libraries use?